docs/group-user-management.rst

```diff
@@ -228,6 +243,35 @@ The administrator or administrator with restricted privileges can add an LDAP us
 
 #. Note that it is advisable to clarify the OMERO group membership situation of the LDAP users soon after their joined OMERO. This can be done for example by adding the new user to their lab group (e.g. ``Lab1``) in OMERO as well and changing the default group of such user in OMERO to be their lab group. See above for how to change the default group of a user. Otherwise, the new LDAP&OMERO users might be importing their data into the ``My Data`` group for some period of time, without realizing the data are not accessible (because ``My Data`` is a private group) to their colleagues in the lab group for cooperative purposes.
 
+*Set up OMERO server to email users*
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+If you are a full administrator or a `restricted administrator <https://docs.openmicroscopy.org/omero/latest/sysadmins/restricted-admins.html>`_ 
```

Comment:
The term 'restricted' admin is used for the first time. This is a bit confusing since you mention 3 categories above: User, Administrator and Administrator with restricted privileges